Triangulation_2/include/CGAL/Constrained_triangulation_plus_2.h

```diff
@@ -773,7 +773,14 @@ insert_subconstraint(Vertex_handle vaa,
   // insert the subconstraint [vaa vbb] 
   // it will eventually be splitted into several subconstraints
 {
-  CGAL_triangulation_precondition( vaa != vbb);
+  std::stack<std::pair<Vertex_handle, Vertex_handle> > stack;
+  stack.push(std::make_pair(vaa,vbb));
+
+  while(! stack.empty()){
+    boost::tie(vaa,vbb) = stack.top();
+    stack.pop();
+    CGAL_triangulation_precondition( vaa != vbb);
+  
```

Comment:
The indentation is not right after the modification. The whole block within the `whilte` must be indented.

Reply:
My comment still holds.